src/CSharp/WebAppNetCore/WebAppNetCore/ConfigurationExtensions.cs

```diff
@@ -90,5 +90,15 @@ public static bool EnableSessionManagement(this IConfiguration configuration)
             }
             return bool.Parse(enableSessionManagement);
         }
+
+        public static bool EnablePostLogout(this IConfiguration configuration)
```

Comment:
Use "return bool.TryParse(enablePostLogout, out value) & value;" for shorter and to avoid exception for wrong input.